projects/1.18/assets/epic-samurais/epicsamurai/lang/zh_cn.json

```
@@ -0,0 +1,127 @@
+    "item.epicsamurai.iron_samurai_chestplate": "铁武士胸甲",
+    "item.epicsamurai.iron_samurai_leggings": "铁武士护腿",
+    "item.epicsamurai.iron_samurai_boots": "铁武士靴",
+    "item.epicsamurai.gold_samurai_helmet": "金武士头盔",
+    "item.epicsamurai.gold_samurai_chestplate": "金武士胸甲",
+    "item.epicsamurai.gold_samurai_leggings": "金武士护腿",
+    "item.epicsamurai.gold_samurai_boots": "金武士靴",
+    "item.epicsamurai.diamond_samurai_helmet": "钻石武士头盔",
+    "item.epicsamurai.diamond_samurai_chestplate": "钻石武士胸甲",
+    "item.epicsamurai.diamond_samurai_leggings": "钻石武士护腿",
+    "item.epicsamurai.diamond_samurai_boots": "钻石武士靴",
+    "item.epicsamurai.netherite_samurai_helmet": "下界合金武士头盔",
+    "item.epicsamurai.netherite_samurai_chestplate": "下界合金武士胸甲",
+    "item.epicsamurai.netherite_samurai_leggings": "下界合金武士护腿",
+    "item.epicsamurai.netherite_samurai_boots": "下界合金武士靴",
+    "item.epicsamurai.red_samurai_helmet": "红宝石武士头盔",
+    "item.epicsamurai.red_samurai_chestplate": "红宝石武士胸甲",
+    "item.epicsamurai.red_samurai_leggings": "红宝石武士护腿",
+    "item.epicsamurai.red_samurai_boots": "红宝石武士靴",
+    "item.epicsamurai.green_samurai_helmet": "翡翠武士头盔",
+    "item.epicsamurai.green_samurai_chestplate": "翡翠武士胸甲",
+    "item.epicsamurai.green_samurai_leggings": "翡翠武士护腿",
+    "item.epicsamurai.green_samurai_boots": "翡翠武士靴",
+    "item.epicsamurai.blue_samurai_helmet": "海蓝宝石武士头盔",
+    "item.epicsamurai.blue_samurai_chestplate": "海蓝宝石武士胸甲",
+    "item.epicsamurai.blue_samurai_leggings": "海蓝宝石武士护腿",
+    "item.epicsamurai.blue_samurai_boots": "海蓝宝石武士靴",
+    "item.epicsamurai.white_samurai_helmet": "银武士头盔",
+    "item.epicsamurai.white_samurai_chestplate": "银武士胸甲",
+    "item.epicsamurai.white_samurai_leggings": "银武士护腿",
+    "item.epicsamurai.white_samurai_boots": "银武士靴",
+    "item.epicsamurai.gray_samurai_helmet": "玛瑙武士头盔",
+    "item.epicsamurai.gray_samurai_chestplate": "玛瑙武士胸甲",
+    "item.epicsamurai.gray_samurai_leggings": "玛瑙武士护腿",
+    "item.epicsamurai.gray_samurai_boots": "玛瑙武士靴",
+    "item.epicsamurai.straw_hat": "草帽",
+    "item.epicsamurai.akaname_spawn_egg": "垢尝刷怪蛋",
+    "item.epicsamurai.enenra_spawn_egg": "烟烟罗刷怪蛋",
+    "item.epicsamurai.oni_spawn_egg": "鬼刷怪蛋",
+    "item.epicsamurai.kitsune_spawn_egg": "狐妖刷怪蛋",
+    "item.epicsamurai.onibi_spawn_egg": "鬼火刷怪蛋",
+    "item.epicsamurai.ruby": "红宝石",
+    "item.epicsamurai.jade": "翡翠",
+    "item.epicsamurai.onyx": "缟玛瑙",
+    "item.epicsamurai.aquamarine": "海蓝宝石",
+    "item.epicsamurai.silver_ingot": "银锭",
+    "item.epicsamurai.raw_silver": "粗银",
+    "item.epicsamurai.silver_nugget": "银粒",
+    "item.epicsamurai.steel_ingot": "钢锭",
+    "item.epicsamurai.steel_nugget": "钢粒",
+    "item.epicsamurai.amethyst_ingot": "紫水晶覆层钢锭",
+    "item.epicsamurai.quartz_ingot": "石英覆层钢锭",
+    "item.epicsamurai.katana": "武士刀",
+    "item.epicsamurai.naginata": "薙刀",
+    "item.epicsamurai.nagamaki": "长卷",
+    "item.epicsamurai.kunai": "苦无",
+    "item.epicsamurai.kama": "忍者镰刀",

```

怎么确定`忍者`这部分的？

---

全篇为日本武士背景，并且游戏中该物品不具有收获作物的功能。

---

我不认为需要加入`忍者`进行补充或强调武器的部分
而且古代大多是农民军，使用镰刀（长或短）作为武器参战很常见
```suggestion
    "item.epicsamurai.kama": "镰刀",
```